infra/blueprint-test/pkg/discovery/config.go

```diff
@@ -49,11 +49,11 @@ func GetTestConfig(path string) (BlueprintTestConfig, error) {
 
 // isValidTestConfig validates a given BlueprintTestConfig
 func isValidTestConfig(b BlueprintTestConfig) error {
-	if b.APIVersion != blueprintTestAPIVersion {
-		return fmt.Errorf("invalid APIVersion %s expected %s", b.APIVersion, blueprintTestAPIVersion)
+	if b.ResourceMeta.APIVersion != blueprintTestAPIVersion {
```

Comment:
Did the linter throw an error for these? `BlueprintTestConfig` embeds `yaml.ResourceMeta` so ` b.APIVersion` should be correct AFAIK since it becomes a promoted field.

Reply:
Yup, valid `go`, however changed for the linter which was raising `typecheck: type <struct> has no field or method <Field on embedded struct from another package>` https://github.com/golangci/golangci-lint/issues/826